src/index.ts

```diff
@@ -149,9 +149,18 @@ export async function main(
       coverageFilePaths = args.file
     }
   }
+  const isNegated = (path: string) => path.startsWith('!')
   coverageFilePaths = coverageFilePaths.concat(await getCoverageFiles(
     args.dir || projectRoot,
-    coverageFilePaths.length > 0 ? coverageFilePaths : coverageFilePatterns(),
+    (() => {
+      const numOfNegatedPaths = coverageFilePaths.filter(isNegated).length
+      
+      if (coverageFilePaths.length > numOfNegatedPaths) {
+        return coverageFilePaths
+      } else {
+        return coverageFilePaths.concat(coverageFilePatterns())
+      }
+    })(),
```

Comment:
The idea here is that, if the user only specifies exclusion filters, the default coverage file patterns are used.